core/types/access_list_tx.go

```diff
@@ -455,13 +455,13 @@ func (tx *AccessListTx) FakeSign(address libcommon.Address) (Transaction, error)
 	cpy.R.Set(u256.Num1)
 	cpy.S.Set(u256.Num1)
 	cpy.V.Set(u256.Num4)
-	cpy.from.Store(address)
+	cpy.StoreFrom(address)
```

Comment:
Rather than wrapping, checking nil should be a part of the accessing code

Reply:
So you prefer like this?
```
if cpy.TransactionMisc == nil {
	cpy.TransactionMisc = &TransactionMisc{}
}
cpy.from.Store(address)
```

I think it's easy to make nil pointer errors if developer forget to check nil. So I wrap store/load methods like this. Can you tell me why you prefer manual nil check?

Reply:
Because it's never intended to be used with a `nil`, and if the developer forgets to check nil, it must be a bug before that point

Reply:
@somnathb1 Got it. I have some questions:
1. Are you talking about only this method(FakeSign)? if so, I agree that it's never called with nil TransactionMisc because it's initialized in copy(). 
2. Or are you talking about every code using the new Load/Store methods in this PR?

I made these new Load/Store methods because I changed the TransactionMisc in TX structs to a pointer, so TransactionMisc won't be initialized automatically when TX instances are initialized. Actually, I saw the nil pointer error before I made these methods.

As you know, `TransactionMisc` is used like cache in the current code, so I believe calling load/store methods before creating it is possible. 
What do you think? Please let me know I'm missing something!

Reply:
All code, in general.
I think, though, that making `TransactionMisc` a pointer isn't the right way to go. It should rather be that all places that make use of "creating" a `xyz_Tx` object should do it with a `store()` call. Thus, this PR, I don't fully agree with

Reply:
@somnathb1 May I ask you why you don't think making TransactionMisc as a pointer is the right way?